src/Components/CreateNewPoi/PoiRating.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { ReactComponent as Starcolor } from '../pictos/Starcolor.svg';
+import { ReactComponent as StarUncolor } from '../pictos/StarUncolor.svg';
+
+const mapStateToProps = state => ({
```

Comment:
You don't need the mapStateToProps now because you don't use datas from the redux state